SurveyAppTests/Tests/Services/Network/Services/AuthenticationServiceSpec.swift

```diff
@@ -0,0 +1,101 @@
+// swiftlint:disable all
+//
+//  AuthenticationServiceSpec.swift
+//  SurveyAppTests
+//
+//  Created by Chananchida F. on 6/17/21.
+//
+
+import Quick
+import Nimble
+import OHHTTPStubs
+
+@testable import SurveyApp
+
+final class AuthenticationServiceSpec: QuickSpec {
+
+    override func spec() {
+
+        var authenticationService: AuthenticationService!
+        var userSession: UserSessionProtocolMock!
+        var api: BaseAPI!
+
+        describe("an AuthenticationService") {
+
+            afterEach {
+                HTTPStubs.removeAllStubs()
+            }
+
+            beforeEach {
+                userSession = UserSessionProtocolMock()
+                api = BaseAPI(userSession: userSession)
+
+                authenticationService = AuthenticationService(api: api, baseURL: Defines.baseURL)
+            }
+
+            // MARK: – authenticateEmail
+            describe("when calling authenticateEmail") {
```

Comment:
a describe shouldn't begin with `when`, it will lead it to become a context
```suggestion
            describe("it's authenticateEmail is called") {
```